src/app/routes/cpsAsset/utils/getAdditionalPageData.js

```diff
@@ -145,14 +146,27 @@ const fetchUrl = async ({ name, path, attachAgent, ...loggerArgs }) => {
   try {
     const agent = attachAgent ? await getAgent() : null;
 
-    return fetchPageData({
-      path,
-      timeout: SECONDARY_DATA_TIMEOUT,
-      agent,
-      ...loggerArgs,
-    })
-      .then(response => validateResponse(response, name))
-      .catch(noop);
+    if (name.indexOf('ecommendations') !== -1){
```

Comment:
Small typo here for `recommendations`.

May need to take into account the other data types that could be cached, `mostRead`, `mostWatched`? Or is this just to test `recommendations`?

Reply:
ah, yes, I realised that some recommendations were Recommendations and some were recommendations and didn't want extra weight of a case insensitive regex, so I cut off first letter 😂 

tbh, couldn't remember if string comparison was case insensitive by default. Just wanted the thing just to work. I tested this by outputting `cache.keys()` and it was definitely using the cache after first load, which is good.

You are right, needs a bit of refactoring/polishing to help it be production ready